pkg/cmd/backuprestore/backupserver.go

```diff
@@ -52,16 +76,61 @@ func (b *backupServer) AddFlags(fs *pflag.FlagSet) {
 }
 
 func (b *backupServer) Validate() error {
+	if !b.enabled {
+		klog.Infof("backup-server is disabled")
+		return nil
+	}
+
+	cronSchedule, err := cron.ParseStandard(b.schedule)
+	if err != nil {
+		pErr := fmt.Errorf("error parsing backup schedule %v: %w", b.schedule, err)
+		klog.Error(pErr)
+		return pErr
```

Comment:
```suggestion
		return fmt.Errorf("error parsing backup schedule %v: %w", b.schedule, err)
```

no need to log here, the root cmd does it already